src/auth/new-routes.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+/*
+HTTPie CLI commands to add some roles:
+echo '{"role":"user", "capabilities":["read"]}' | http :3000/roles
+echo '{"role":"editor", "capabilities":["create", "read", "update"]}' | http :3000/roles
+echo '{"role":"admin", "capabilities":["create","read","update","delete"]}' | http :3000/roles
+echo '{"role":"superuser", "capabilities":["create","read","update","delete","superuser"]}' | http :3000/roles
+
+
+HTTPie CLI commands to add some users:
+echo '{"username":"usher", "password":"usher", "role":"user"}' | http :3000/signup
+echo '{"username":"eddie", "password":"eddie", "role":"editor"}' | http :3000/signup
+echo '{"username":"addie", "password":"addie", "role":"admin"}' | http :3000/signup
+echo '{"username":"susie", "password":"susie", "role":"superuser"}' | http :3000/signup
+
+superuser guy key:
```

Comment:
I was able to get your superuser to work, however, I could not get your admin, user or editor roles to work with the correct capabilities.